.github/workflows/deploy.yaml

```diff
@@ -104,14 +104,14 @@ jobs:
           <settings>
             <servers>
               <server>
-                <id>ossrh</id>
-                <username>${{ env.OSSRH_USERNAME }}</username>
-                <password>${{ env.OSSRH_PASSWORD }}</password>
+                <id>central</id>
+                <username>${{ env.CENTRAL_USERNAME }}</username>
```

Comment:
As an FYI, I left the old variable names the same in the `fusionauth-developer`. I don't think it hurts to leave the names in `fusionauth-developer` since they are just input variables used to replace in the template, but up to you if you want to change them to match these.